### DISCUSSION TITLE
Controller for Player2 on RG351MP?

### DISCUSSION THREAD

**Discussion by gabuzom** [351ELEC/351ELEC#935] [General]
Hey there!

Has anyone ever succeeded in connecting a generic gamepad on the OTG port of the RG351MP and enabling it for player2 support in retroarch?

If so, how did you do it?

Thanks!

**Comment by enfermero-dev** [comment]
~~I tried with mine but failed. Also the only device i could connect to my RG351MP is the included wifi dongle. ~~

**Comment by enfermero-dev** [comment]
I've testing an 'xbox to ant console' dongle found on ali and it works! Requires some config tweaking to have it work for P1 without getting the native controller disconnected (and make the os uncontrollable) but for now i can control the menu. ![image](https://user-images.githubusercontent.com/126836625/223624234-631e62c7-5d12-4884-abc5-a21ec6fc93f5.jpeg)

The dongle make an xbox one bluetooth controller usable on my nintendo switch (and other consoles) mimicking an usb wired controller. 

RG351MP detects it and even it says "Microsoft X-Box 360 pad". I will test my xbox one bt ct wired to the RG351MP. 

... and it was detected. 
![image](https://user-images.githubusercontent.com/126836625/223625081-28cd2e0c-6ee7-4266-9aba-3d39ab3939d3.jpeg)

It reads "Microsoft X-Box One S pad". 

Now we need tv output and we are ready to go 🤪🤪. Hope this can help, ignore the red apparatus connected to the console, its a Bluetooth audio transmitter.

**Comment by enfermero-dev** [comment]
I just noticed: you simply connect the controller when in menu and it will ask you for binding from there.

**Comment by gabuzom** [comment]
Wow! Will try again with latest versions. I have a USB (wired) 8bitdo controler that is recognize as Xbox 360... and saw nothing before (in previous versions) good news!! <3
